### PR TITLE
[Sprint 7] inital steps towards rock-wallet-branding

### PR DESCRIPTION
**Pull request by dabitdev**

### CODE REVIEW

File: brd-android/app/src/main/res/layout/controller_home.xml [hunk 1]
```diff
@@ -42,24 +42,15 @@
...
                 android:id="@+id/total_assets_label"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_alignParentEnd="true"
+                android:layout_centerHorizontal="true"
                 android:layout_marginTop="50dp"
                 android:layout_marginEnd="@dimen/activity_vertical_margin"
                 android:fontFamily="@font/mobile_font_medium"
-                android:text="@string/HomeScreen.totalAssets"
+                android:text="Balance"
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/controller_home.xml]: Hardcoded text

File: brd-android/app/src/main/res/layout/controller_intro.xml [hunk 2]
```diff
@@ -28,7 +37,7 @@
         android:layout_marginLeft="16dp"
         android:layout_marginRight="16dp"
         android:padding="10dp"
-        android:text="@string/Onboarding.getStarted"
+        android:text="Create new wallet"
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/controller_intro.xml]: Hardcoded text

File: brd-android/app/src/main/res/layout/controller_intro.xml [hunk 3]
```diff
@@ -66,14 +75,14 @@
         android:layout_marginBottom="32dp"
         android:fontFamily="@font/mobile_font_book"
         android:lineSpacingMultiplier="1.3"
-        android:text="@string/OnboardingPageOne.title"
+        android:text="Empowering you in the emerging digital asset economy"
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/controller_intro.xml]: Hardcoded text

File: brd-android/app/src/main/res/layout/controller_welcome.xml [hunk 4]
```diff
@@ -61,19 +68,18 @@
...
         android:layout_height="wrap_content"
         android:layout_marginLeft="8dp"
-        android:layout_marginTop="32dp"
+        android:layout_marginTop="40dp"
         android:layout_marginRight="8dp"
-        android:layout_marginBottom="32dp"
         android:fontFamily="@font/mobile_font_book"
         android:lineSpacingMultiplier="1.3"
-        android:text="@string/OnboardingPageOne.title"
+        android:text="Empowering you in the emerging digital asset economy"
```
Comment (roknovak) [inline brd-android/app/src/main/res/layout/controller_welcome.xml]: Hardcoded text

**Comment by dabitdev** [comment]
@roknovak I will address these hardcoded texts asap

**Review by roknovak — APPROVED** [review]

**Comment by dabitdev** [comment]
Hi @roknovak 

could you have look at why these tests are failing in the CI?

```
com.fabriik.buy.ui.features.orderpreview.OrderPreviewViewModelTest > onNetworkInfoClicked_verifyEffect FAILED
    java.lang.AssertionError at OrderPreviewViewModelTest.kt:102

com.fabriik.buy.ui.features.orderpreview.OrderPreviewViewModelTest > onCreditInfoClicked_verifyEffect FAILED
    java.lang.AssertionError at OrderPreviewViewModelTest.kt:91

com.fabriik.buy.ui.features.orderpreview.OrderPreviewViewModelTest > onSecurityCodeInfoClicked_verifyEffect FAILED
    java.lang.AssertionError at OrderPreviewViewModelTest.kt:113
```

**Comment by dabitdev** [comment]
Something happened with the new strings I will close this for now

**Pull request closed without merging**